Git Formats/Git Diff.sublime-syntax

```diff
@@ -0,0 +1,567 @@
+%YAML 1.2
+---
+# https://www.sublimetext.com/docs/syntax.html
+# https://git-scm.com/docs/git-format-patch
+# https://git-scm.com/docs/diff-format
+name: Git Diff
+scope: source.diff.git
+version: 2
+extends: Packages/Diff/Diff.sublime-syntax
+
+file_extensions:
+  - diff.eml
+  - patch.eml
+  - diff.mbox
+  - patch.mbox
```

Comment:
Most likely linux kernel hackers will be the only ones who benefit from opening their patches with this syntax directly.

That's the first time I've learned about these extensions. 

Creating patches with git or Sublime Merge just create `.patch` files, which are opened with _Diff_.

Reply:
I don't remember exactly. I may have invented these extensions. If I just say `patch` here, what happens when a user opens a `patch` file? Would anyone unexpectedly get the Git one?

Reply:
IIRC syntaxes loaded later, overwrite extension assignments.

Reply:
Not sure, how common they are. At least appear to be mailing-list-workflow-specific.

I can't judge, whether they are useful or may cause confusion in future.